Zend/zend_vm_gen.php

```diff
@@ -1779,25 +1841,42 @@ function gen_executor($f, $skl, $spec, $kind, $executor_name, $initializer_name)
 				case "ZEND_VM_DISPATCH":
 				  // Emit code that dispatches to opcode handler
 					switch ($kind) {
+						case ZEND_VM_KIND_SWITCH:
+							out($f, $m[1]."dispatch_handler = OPLINE->handler;\nzend_vm_dispatch:\n".$m[1]."switch ((int)(uintptr_t)dispatch_handler)".$m[3]."\n");
+							break;
+						case ZEND_VM_KIND_GOTO:
+							out($f, $m[1]."goto *(void**)(OPLINE->handler);".$m[3]."\n");
+							break;
+						case ZEND_VM_KIND_HYBRID:
+							out($f,"#if (ZEND_VM_KIND == ZEND_VM_KIND_HYBRID)\n");
+							out($f, $m[1]."HYBRID_SWITCH();".$m[3]."\n");
```

Comment:
The position of this code is pretty weird. Why is it not generated directly above the HYBRID_CASEs?